src/chart/bar/BarView.ts

```diff
@@ -982,6 +982,9 @@ function updateStyle(
 
     if (isPolar) {
         const position = seriesModel.get(['label', 'position']);
+        if (position === 'middle') {
+            el.textConfig.inside = true;
```

Comment:
Should always set `inside` property to avoid current configuration leaking to the next render.